taskcluster/ci/toolchain/kind.yml

```diff
@@ -1,54 +1,30 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
 ---
 loader: taskgraph.loader.transform:loader
+
+kind-dependencies:
+    - fetch
+
 transforms:
     - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.cached_tasks:transforms
     - taskgraph.transforms.task:transforms
 
+
 job-defaults:
-    worker-type: always-optimized
     run:
         using: index-search
+    run-on-projects: []
+    worker-type: always-optimized
```

Comment:
👍

Done. There is no `job-defaults` in `kind.yml` anymore.

Reply:
It might be worthwhile revisit this in the future. If we ever have more locally defined toolchains, I suspect they'll all want the defaults currently in `android.yml`, so it might make sense to define them here, and have `gecko-derived.yml` override those as appropriate.